openshift/e2e-tests-openshift.sh

```diff
@@ -131,7 +131,7 @@ function create_test_resources_openshift() {
 function resolve_resources(){
   local dir=$1
   local resolved_file_name=$3
-  for yaml in $(find $dir -name "*.yaml" -mindepth 1 -maxdepth 1); do
+  for yaml in $(find $dir -name "*.yaml"); do
```

Comment:
Having `-mindepth` and `-maxdepth` there means that monitoring will not be installed - this is good to save a bit of resources.